overlord/servicestate/quota_control_test.go

```diff
@@ -261,7 +269,14 @@ func (s *quotaControlSuite) TestCreateQuotaPreseeding(c *C) {
 	snaptest.MockSnapCurrent(c, testYaml, s.testSnapSideInfo)
 
 	// now we can create the quota group
-	err := servicestate.CreateQuota(s.state, "foo", "", []string{"test-snap"}, quantity.SizeGiB)
+	qc := servicestate.QuotaControlAction{
+		Action:      "create",
```

Comment:
nitpick: afair all the quota* helpers don't look at Action, so maybe in these tests it should be left out?

Reply:
I think it's more realistic to leave it in, but yes you are right that it isn't needed